src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
Add an empty line between sibling blocks, as requested in [Code Style Rules](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

Reply:
You're absolutely right! I'll fix that right away and get back to you!